client/src/app/core/core-services/websocket.service.ts

```diff
@@ -183,13 +189,17 @@ export class WebsocketService {
             this.zone.run(() => {
                 this.websocket = null;
                 this._connectionOpen = false;
+                // 1000 is a normal close, like the close on logout
                 if (event.code !== 1000) {
                     console.error(event);
                     // Do not show the message snackbar on the projector
                     // tests for /projector and /projector/<id>
                     const onProjector = this.router.url.match(/^\/projector(\/[0-9]+\/?)?$/);
-                    // 1000 is a normal close, like the close on logout
-                    if (!this.connectionErrorNotice && !onProjector) {
+                    if (this.retryCounter <= 3) {
```

Comment:
`3` was used twice. Put in an own (private readonly) variable `nrOfRetryTries` (or similar).
Do that in another PR.

Reply:
Yes, this will give mergeconfilict with an upcoming PR. I'll do this there.